### PR TITLE
Update installation instructions for latest ubuntu version

### PR DESCRIPTION
**Pull request by tsubery**
This installation procedure works for ubuntu 16.04

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: install.markdown [hunk 1]
```diff
@@ -44,7 +44,7 @@ If your distribution contains an old Elixir/Erlang version, see the sections bel
   * FreeBSD
     * From ports: `cd /usr/ports/lang/elixir && make install clean`
     * From pkg: `pkg install elixir`
-  * Ubuntu 12.04 and 14.04 / Debian 7
+  * Ubuntu 12.04/14.04/16.04 or Debian 7
```
Comment (eksperimental) [inline install.markdown]: this will most likely work for 12.10, 14.10 and 16.10,
so i'm wondering if we should just mention from 12.04 onwards...

**Comment by tsubery** [comment]
13 &15 have passed End of life so they won't likely work. We could somehow
phrase this exception

**Comment by eksperimental** [comment]
sorry, but why wouldn't they work?

**Comment by tsubery** [comment]
I actually tried spawning a 13.04 vm and test it out before submitting the pull request. After few minutes of trying to figure out why apt-get can't find the necessary dependencies I realized it was long since this version passed End of Life. I can imagine other users going through the same process if it specifically claimed to be supported.

**Comment by eksperimental** [comment]
well, it does not necessarily mean that software stops working after its life support's expired. but it will make our lives easier if we just limit ourselves to give advice to whatever is still being supported.

**Pull request merged**